Add examples to make_list (#228)
MicrosoftDocs/dataexplorer-docs: data-explorer/kusto/query/makelist-aggfunction.md

> [!TIP]
> Use the [`mv-apply`](./mv-applyoperator.md) operator to create an ordered list by some key. See examples [here](./mv-applyoperator.md#using-the-mv-apply-operator-to-sort-the-output-of-makelist-aggregate-by-some-key).

## Examples

> ### One Column

Reviewer: ```suggestion
### One column
```

> Most simple example is to make a list out of a single column:

Reviewer: ```suggestion
The simplest example is to make a list out of a single column:
```

```kusto
let shapes = datatable (name: string, sideCount: int)
[
    "triangle", 3,
    "square", 4,
    "rectangle", 4,
    "pentagon", 5,
    "hexagon", 6,
    "heptagon", 7,
    "octogon", 8,
    "nonagon", 9,
    "decagon", 10
];
shapes

> | summarize mylist=make_list(name)

Reviewer 2: ```suggestion
| summarize mylist = make_list(name)
```

```

|mylist|
|---|
|["triangle","square","rectangle","pentagon","hexagon","heptagon","octogon","nonagon","decagon"]|

### Using the 'by' clause

> We can group by:

Reviewer: ```suggestion
In the following query, you group using the `by` clause:
```

```kusto
let shapes = datatable (name: string, sideCount: int)
[
    "triangle", 3,
    "square", 4,
    "rectangle", 4,
    "pentagon", 5,
    "hexagon", 6,
    "heptagon", 7,
    "octogon", 8,
    "nonagon", 9,
    "decagon", 10
];
shapes

> | summarize mylist=make_list(name) by isEvenSideCount = not(tobool(sideCount % 2))

Reviewer 2: ```suggestion
| summarize mylist = make_list(name) by isEvenSideCount = sideCount % 2 == 0
```

```

|mylist|isEvenSideCount|
|---|---|
|false|["triangle","pentagon","heptagon","nonagon"]|
|true|["square","rectangle","hexagon","octogon","decagon"]|

### Packing a dynamic object

> We can [pack](./packfunction.md) a dynamic object in a column before making a list out of it:

Reviewer: ```suggestion
You can [pack](./packfunction.md) a dynamic object in a column before making a list out of it, as seen in the following query:
```

```kusto
let shapes = datatable (name: string, sideCount: int)
[
    "triangle", 3,
    "square", 4,
    "rectangle", 4,
    "pentagon", 5,
    "hexagon", 6,
    "heptagon", 7,
    "octogon", 8,
    "nonagon", 9,
    "decagon", 10
];
shapes

> | extend d=pack("name", name, "sideCount", sideCount)

Reviewer 2: ```suggestion
| extend d = pack("name", name, "sideCount", sideCount)
```

> | summarize mylist=make_list(d) by isEvenSideCount = not(tobool(sideCount % 2))

Reviewer 2: ```suggestion
| summarize mylist = make_list(d) by isEvenSideCount = sideCount % 2 == 0
```
